services/app/src/hooks/useChessBoard.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import { numberOfRows, numberOfColumns } from '../constants/board';
+
+const alphabetArray = [...'abcdefghijklmnopqrstuvwxyz'];
+
+export const useChessBoard = () => {
+  const [board, setBoard] = useState(
+    [...Array(numberOfRows + 1).keys()].map((_row, rowIndex) =>
+      [...Array(numberOfColumns + 1).keys()].map((_tile, tileIndex) => ({
+        attribute:
+          tileIndex === 0 && rowIndex !== numberOfRows
+            ? numberOfColumns - rowIndex
```

Comment:
why not get rid of one of the ternaries and explicitly have a row for numbers?
```
[
  ...Array(numberOfRows).keys().map(() => etc),
  [...[...Array(numberOfRows).keys()].map(k => k + 1))
]
```
Or just
```
[
  ...Board stuff,
  [1, 2, 3, 4, 5, 6, 7, 8]
]
```

Reply:
Same with the symbol for columns
just add the symbol as an element before spreading the rest of the board stuff in and you won't need a 3 story ternary.

Reply:
My bad: row for symbols, column for numbers. But the point is the same - have explicit elements to simplify the ternary.